src/parser/jobs/smn/Aetherflow.js

```diff
@@ -28,30 +28,45 @@ export default class Aetherflow extends Module {
 		'suggestions',
 	]
 
-	_badFesters = []
+	_badBanes = []			// In case we ever want to check for Banes where 1 or 0 DoTs are spread
```

Comment:
Avoid using tabs for alignment - it throws things out if people are using a different tabwidth.